private/buf/cmd/buf/command/beta/registry/commit/commitlist/commitlist.go

```diff
@@ -124,6 +124,7 @@ func run(
 		flags.PageSize,
 		flags.PageToken,
 		flags.Reverse,
+		false,
```

Comment:
We should set this to true some time before `v1.0.0`, but I don't want to add that decision to the scope of this PR.